fsi/solutions/unifiedCustomerProfile/readme.md

```diff
@@ -0,0 +1,111 @@
+
+For more details on how to grant users access to an environment please visit [this site](https://docs.microsoft.com/en-us/power-platform/admin/add-users-to-environment). For more details about security roles, please visit [this site](https://docs.microsoft.com/power-platform/admin/database-security).
+
+## Features of unified customer profile solution
+
+The following sections will walkthrough the various features of the unified customer profile solution.
+
+### Customer Summary
+
+The customer summary tab shows the most relevant information about a customer. Following details are provided and are allowed to be edited in the summary tab:
+
+* *Customer snapshot*: personal information such as name, address, demographics and branch information, preferred channel of communication (can be extended with chrun score, and other information)
+* *Life events*: customer's and family's past and future life milestones
+* *Financial holdings*: Accounts, investments, loans, assets, liabilities, credit lines, long-term savings
+* *Cards*: Credit & debit cards held by the customer, status of cards, activation state
+* *Main household*: Household income, total assets, total liabilities
+* *Customer intelligence*: Churn risk (if customer intelligence churn model has been trained and setup)
+
+### Connections
+
+The connections tab provides a summarized view of relationships to other customers and group membership. Following details are shown and can be edited by the user:
+
+* *Customer groups*: Group memberships such as customer's household, role in group
+* *Selected customer group*: Detailed group information includes group address, total income, total assets, total liabilities, shared financial holdings
+* *Relationships*: Relationship to other customers including private and professional relationships
+
+### Financial Holdings
+
+The financial holdings tab provides more detailed insights about the financial holdings shown in the summary tab. Following information can be viewed in this section:
+
+* *Financial holdings*: Financial holdings name, category, type, value, instruments, alerts for savings or loans nearing the maturity date
+* *Currency treatment*: Detailed fianncial holdings overview in the original currency and teh default currency
+
+## Planning guidelines for unified customer profile
+
+This section provides prescriptive guidance with design considerations and recommendations across the critical design areas for unified customer profile for the teams that will deploy and manage the capabilities within the Microsoft Cloud for Financial Services.
+
+> Note: The design areas for unified customer profile are lightweight as the critical design areas, considerations and recommendations should be addressed in the Power Platform architecture itself. This is primarily because the unified customer profile scenario and its dependencies are D365 applications that must only be sequenced and deployed into existing environments.
```

Comment:
Consider rewording? 

"United customer profile is based on Dynamics 365 and Power Platform, hence it inherits design decisions and implementation details from the underlying foundational building blocks."

Reply:
Same a above

Reply:
Same as above - let's take it offline. I think we can make verbiage more concise. 
Maybe I am just being pednatic :)